### ISSUE TITLE
Accessibility: Built-in VSCode webviews  fails with WCAG 2.1 SC 2.4.2 Page Titled (Level A) - Missing <title> element in <head> element

### ISSUE THREAD

**Issue by RushikeshTrivedi** [microsoft/vscode#321656]
Does this issue occur when all extensions are disabled?: Yes

VS Code Version: 1.119.0
OS Version: Windows 11

**WCAG Requirement:**

https://www.w3.org/WAI/WCAG22/Understanding/page-titled

**Steps to Reproduce:**

Open VSCODE
Select Extension
Run accessibility checker on it

**Rationale:**
This requirement ensures descriptive titles are provided. Descriptive titles help all users orient themselves when navigating through content or when moving between multiple applications or documents.

Screen readers announce titles before other content. If titles are missing, screen reader users must explore more content to determine the purpose. Conversely, if titles are identical for multiple pages, or repeat the same information before providing unique information, a user will hear repetitive content before understanding context.

The title should:

Succinctly identify the subject
Make sense when read out of context, for example by a screen reader, in a site map, list of search results, window title, or on a browser tab title
If the page or document belongs to a collection, the title should:

List the most unique identifying information first
Be unique within the collection

<img width="3289" height="1641" alt="Image" src="https://github.com/user-attachments/assets/2ac2a2d9-376f-4866-bce9-f42c1aed56f1" />

<img width="3449" height="1274" alt="Image" src="https://github.com/user-attachments/assets/7bf9c174-a98f-4daf-9c9b-21e07a25952c" />
